src/main.c

```diff
@@ -521,6 +546,72 @@ bool handleLayer4SpecialCases(KBDLLHOOKSTRUCT keyInfo)
 		// extended flag (bit 0) is necessary for selecting text with shift + arrow
 		keybd_event(mappingTableLevel4Special[keyInfo.scanCode], bScan, dwFlagsFromKeyInfo(keyInfo) | KEYEVENTF_EXTENDEDKEY, 0);
 
+	mappingTable[16] = VK_PRIOR;
```

Comment:
Dieser Teil ist zwischenzeitlich durch einen anderen Pull Request in eine neue Funktion namens `initLevel4SpecialCases` (Zeile 151ff in deiner Version) gewandert. (Es bringt ja nichts, diese Arrays bei jedem Ebene4-Tastendruck neu zu definieren, da sie sich eh nicht mehr ändern.) Das heißt, `mappingTable` sollte hier nicht mehr beschrieben werden. Der Teil, der das Numpad betrifft, sollte auch in der Funktion `initLevel4SpecialCases` stehen. (Da heißt das Array dann `mappingTableLevel4Special`).

Reply:
ja genau, das habe ich jetzt angepasst